### PR TITLE
Add support for python virtualenv

### PR DESCRIPTION
**Pull request by fgandellini**
I added the support for python virtualenv, it is placed to the far right of the prompt, surrounded by parenthesis.

slimline_venv

I also added a couple of environment variables to enable/disable the feature and change the color.

Let me know if you like this! :smiley:

### CODE REVIEW

**Comment by fgandellini** [comment]
I found a typo calling color variables and forgot to update the TOC with the "Python Virtualenv" section. :stuck_out_tongue_closed_eyes: All fixed now! :sunglasses:

File: slimline.zsh [hunk 1]
```diff
@@ -106,6 +112,11 @@ prompt_slimline_set_rprompt() {
   if [[ -n "${_prompt_slimline_git_output:-}" ]]; then
     RPROMPT+="${RPROMPT:+ }${_prompt_slimline_git_output}"
   fi
+
+  # add virtualenv (if active)
+  if (( ${SLIMLINE_DISPLAY_VIRTUALENV:-1} )); then
+    RPROMPT+="${RPROMPT:+ }$(prompt_slimline_virtualenv)"
```
Comment (mengelbrecht) [inline slimline.zsh]: This adds a trailing space to the prompt if `VIRTUAL_ENV ` is empty. We can store it in a variable first and add it only if it is not empty:
```shell
local virtual_env="$(prompt_slimline_virtualenv)"
if [[ -n "${virtual_env}" ]]; then
  RPROMPT+="${RPROMPT:+ }${virtual_env}"
fi
```

File: slimline.zsh [hunk 2]
```diff
@@ -74,6 +74,12 @@ prompt_slimline_cwd() {
   echo "%F{${cwd_color}}%3~%f "
 }
 
+prompt_slimline_virtualenv() {
+  local parens_color='008'
```
Comment (mengelbrecht) [inline slimline.zsh]: This should be customizable via eg. `SLIMLINE_VIRTUALENV_PARENS_COLOR` (and added to the readme).

**Review by mengelbrecht — CHANGES_REQUESTED** [review]
Thanks for your work! With the two small changes, it is ready to go 👍

File: slimline.zsh (hunk 2 above)
Comment (fgandellini) [reply slimline.zsh]: Yep! 
I would like to keep the color definition aligned with the text-color-codes of the project, unfortunately the "gray" color is not available in zsh, so I think that I'll go with "white" as default.

Comment (mengelbrecht) [reply slimline.zsh]: Sounds good! :-)

File: slimline.zsh (hunk 1 above)
Comment (fgandellini) [reply slimline.zsh]: You're right! I'll send the fix ASAP.

**Review by mengelbrecht — APPROVED** [review]

**Comment by mengelbrecht** [comment]
Thanks! Looks nice.

**Comment by fgandellini** [comment]
Wow! :smiley:
Thanks for your work, I love slimline! :heart_eyes:

**Pull request merged**